vmdb/app/helpers/application_helper.rb

```diff
@@ -1002,8 +1002,10 @@ def build_toolbar_hide_button(id)
         return true if !@record.is_available?(:shutdown_guest)
       when "vm_guest_restart", "instance_guest_restart"
         return true if !@record.is_available?(:reboot_guest)
-      when "vm_migrate", "vm_reconfigure"
-        return true if @record.vendor.downcase == "redhat"
+      when "vm_migrate"
+        return true unless @record.supports_migrate?
+      when "vm_reconfigure"
+        return true unless @record.supports_reconfigure?
```

Comment:
@Fryguy is_available? method take a parameter and calls validate_xx methods that do something similar to what supports_migrate? does, if you want i can change this code to go thru is_available? method, let me know.